community/index.tt

```diff
@@ -288,6 +288,11 @@
       <p>Moderates participation on official community platforms.</p>
       <a class="button -primary" href="[%root%]community/teams/moderation.html">Read more</a>
 
+      [% PROCESS svg path="site-styles/assets/gfx-gpu.white.svg" %]
+      <h3>CUDA Team</h3>
+      <p>Making NixOS and Nixpkgs the first choice for users of CUDA-accelerated software.</p>
```

Comment:
```suggestion
      <p>Making Nix and NixOS the first choice for users of CUDA-accelerated software.</p>
```

I suppose this is what we really want to say here. You cannot use Nixpkgs without Nix (or some clone).

Reply:
Fixed in newest